app/assets/stylesheets/components/_button.scss

```diff
@@ -0,0 +1,11 @@
+.button {
```

Comment:
so right here it would also need to be changed to show-page-button, or something similar whatever you want

Reply:
OK. I can rename it